caveclient/frameworkclient.py

```diff
@@ -339,6 +341,7 @@ def __init__(
         - `client.info` is an `InfoServiceClient` (see [client.info](../client_api/info.md))
         - `client.l2cache` is an `L2CacheClient` (see [client.l2cache](../client_api/l2cache.md))
         - `client.materialize` is a `MaterializationClient` (see [client.materialize](../client_api/materialize.md))
+        - `client.skeleton` is a `SkeletonClient` (see [client.skeleton](../client_api/skeleton.md))
```

Comment:
for this to work and for the docs to render, would need to add this file. can mirror what is here https://github.com/CAVEconnectome/CAVEclient/tree/master/docs/client_api

Reply:
Noted.